app/src/main/java/helium314/keyboard/keyboard/clipboard/ClipboardHistoryView.kt

```diff
@@ -240,6 +242,22 @@ class ClipboardHistoryView @JvmOverloads constructor(
         }
     }
 
+    override fun onLongClick(view: View): Boolean {
+        val tag = view.tag
+        if (tag is ToolbarKey) {
+            val longClickCode = getCodeForToolbarKeyLongClick(tag)
+            if (longClickCode != KeyCode.UNSPECIFIED) {
+                keyboardActionListener?.onCodeInput(
+                    longClickCode,
+                    Constants.SUGGESTION_STRIP_COORDINATE,
+                    Constants.SUGGESTION_STRIP_COORDINATE,
```

Comment:
Is there a specific reason you use `SUGGESTION_STRIP_COORDINATE` here? If not, I would suggest to use `NOT_A_COORDINATE` like it's done `onClick`.

Or if there is a good reason to use `SUGGESTION_STRIP_COORDINATE`, maybe it should be used in `onClick` too?

Reply:
but isn't it already using 'SUGGESTION_STRIP_COORDINATE' in onClick? I remember copying the line from there

Reply:
This is onClick:

```kotlin
    override fun onClick(view: View) {
        when (view) {
            alphabetKey, spacebar, deleteKey -> {
                keyboardActionListener?.onCodeInput(view.tag as Int,
                    Constants.NOT_A_COORDINATE, Constants.NOT_A_COORDINATE, false)
                keyboardActionListener?.onReleaseKey(view.tag as Int, false)
            }
        }
        val tag = view.tag
        if (tag is ToolbarKey) {
            val code = getCodeForToolbarKey(tag)
            if (code != null) {
                keyboardActionListener?.onCodeInput(code, Constants.NOT_A_COORDINATE, Constants.NOT_A_COORDINATE, false)
                return
            }
            if (tag == ToolbarKey.CLEAR_CLIPBOARD)
                clipboardHistoryManager?.clearHistory()
        }
    }
```

Reply:
Right sorry I copied it from the SuggestionStripView, did not realize NOT_A_COORDINATE is used for the ClipboardHistoryView. Will fix this. Thanks